cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/constraint/LastRunConstraint.java

```diff
@@ -30,19 +30,19 @@
  */
 public class LastRunConstraint extends ProtoConstraint.LastRunConstraint implements CheckableConstraint {
 
-  public LastRunConstraint(long millisSinceLastRun) {
-    super(millisSinceLastRun);
+  public LastRunConstraint(long secsSinceLastRun) {
```

Comment:
I think the field in `ProtoConstraint.LastRunConstraint` should still be milliseconds, though. It matters, since users would see this value in REST API, when getting schedules or even when put/updating schedules.